services/convex/docker-entrypoint.sh

```
@@ -0,0 +1,515 @@
+  unset NODE_EXTRA_CA_CERTS 2>/dev/null || true
+fi
+
+# ============================================================================
+# Database wait (shared with platform/crawler/rag; already handled by
+# compose depends_on, but we double-check for safety)
+# ============================================================================
+wait_for_port() {
+  local host="$1" port="$2" timeout="${3:-60}" name="${4:-Service}"
+  log_info "Waiting for ${name} at ${host}:${port}..."
+  local counter=0
+  while ! (echo > /dev/tcp/${host}/${port}) >/dev/null 2>&1; do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)